express/src/controllers/clients.ts

```diff
@@ -23,7 +36,7 @@ export const showClient: RequestHandler = async (req, res) => {
         serviceName: serviceName,
         updatedField: req.session.updatedField,
         redirectUrls: redirectUrls,
-        userAttributesRequired: client.scopes.join(", "),
+        userAttributesRequired: userAttributes.join("<br> "),
```

Comment:
```suggestion
        userAttributesRequired: userAttributes.join("<br> "),
```
This should be generated in the view, not in the controller. The controller doesn't know about the presentation layer or HTML.